### ISSUE TITLE
Fetching live camera stream

### ISSUE THREAD

**Issue by AryanSethi** [gozfree/gear-lib#77]
Can I use `libjpeg-ex` or `libavcap: audio/video capture api (v4l2/uvc/esp32/dshow)` to fetch live camera stream from my web cam in linux ?

**Comment by gozfree** [comment]
yes, you can use libavcap to capture audio/video frame in two methods:
"avcap_query_frame": sync call with one-by-one frame
"avcap_start_stream": with async on_stream callback to get stream
refer to test_libavcap.c

**Comment by gozfree** [comment]
libjpeg-ex is wrapper of libjpeg, aim to convert easily between yuv and jpeg.
